src/Persistence_Array.php

```diff
@@ -365,6 +365,17 @@ public function applyConditions(Model $m, Action\Iterator $q)
                     $cond[1] = $this->typecastSaveField($cond[0], $cond[1]);
                 }
                 $q->where(is_string($cond[0]) ? $cond[0] : $cond[0]->short_name, $cond[1]);
+            } elseif(count($cond) == 3) {
```

Comment:
what if we have `$cond = ['field', '>', 123]` condition ?

Reply:
there is no Iterator::> (method) -> there will be an error